_posts/2017-10-26-totw-64.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Tip of the Week #64: Raw String Literals"
+layout: tips
+sidenav: side-nav-tips.html
+published: draft
+permalink: tips/64
+type: markdown
+order: 064
+---
+
+## Change "published:true" to post actual tip
+
+Originally posted as totw/64 on 2013-12-09
```

Comment:
I've been using "Dec 9, 2013" format (same for updated date).

Reply:
The American date format is intolerable. Can we avoid it?

Reply:
We can do whatever we want. Certainly the American format is bad for anything simply numeric (and we can't use it for the filename, fwiw), but I do actually prefer "Dec 9, 2013" to "2012-12-09". We should change every date to match the latter then (including here).